sentry.gradle

```diff
@@ -153,6 +154,32 @@ gradle.projectsEvaluated {
               enabled true
           }
 
+          def modulesTask = tasks.create(nameModulesTask, Exec) {
+              description = "collect javascript modules from bundle source map"
+              group = 'sentry.io'
+
+              workingDir reactRoot
+
+              def collectModulesScript = project.hasProperty('sentry.collectModulesScript')
+                ? file(project.getProperty('sentry.collectModulesScript')).getAbsolutePath()
+                : "$reactRoot/node_modules/@sentry/react-native/dist/js/tools/collectModules.js"
+              def modulesOutput = "$reactRoot/android/app/src/main/assets/modules.json"
```

Comment:
@markushi Is there a cleaner way of adding a file to the app asset than creating it in `android/app/src/main/assets`?

Reply:
H: Whats the average size of the `modules.json`?
If app size is a concern for some folks, we need to offer a way to disable this feature.
We could add an opt-out via Gradle config, See the usage of `flavorAware`.

Reply:
What you usually do on Android is to define extra [source sets](https://developer.android.com/studio/build/build-variants#sourcesets). There should be one for asset files as well, like `android.sourceSets.main.assets.srcDirs += ['path/to/your/folder]`. But I'm not sure if you have access to `android` in this context.

Reply:
@marandaneto The one from our sample app is 1 KB.